deployments/zksyncmainnet_addresses.json

```diff
@@ -18,6 +18,7 @@
     "RewardsDistributorImpl": "0x1A363085a85a21361c97ee9eeF07a8e61cc0D8e2",
     "RewardsDistributor_Core_0": "0x7C7846A74AB38A8d554Bc5f7652eCf8Efb58c894",
     "RewardsDistributor_Core_0_Proxy": "0x7C7846A74AB38A8d554Bc5f7652eCf8Efb58c894",
+    "TwoKinks_base0bps_slope1500bps_kink8000bps_slope29000bps_base20bps_kink29000bps_jump30000bps": "0x61E98E6a1F37649543156DC995E0dfe466B31a2e",
```

Comment:
It has to be verified

Reply:
https://explorer.zksync.io/address/0x61E98E6a1F37649543156DC995E0dfe466B31a2e#contract